cmd/mic/main.go

```diff
@@ -98,7 +103,10 @@ func main() {
 	config.Burst = int(clientQPS)
 	glog.Infof("Client QPS set to: %v. Burst to: %v", config.QPS, config.Burst)
 
-	micClient, err := mic.NewMICClient(cloudconfig, config, forceNamespaced, syncRetryDuration, &leaderElectionCfg, enableScaleFeatures, createDeleteBatch)
+	immutableUserMSIsList := strings.Split(immutableUserMSIs, ",")
+	glog.Infof("immutable identities are %v", immutableUserMSIsList)
```

Comment:
nit0: Can you please make this conditional, If the parameter is given, then only print. 
nit1: Can we have the first letter with caps.